### PR TITLE
update emsdk to 3.1.47 [don't merge - perf regression]

### PR DESCRIPTION
**Pull request by guschmue**

### CODE REVIEW

**Comment by fs-eire** [comment]
Need to update cgmanifest.json by cgmanifests\generate_cgmanifest.py

File: cmake/onnxruntime_webassembly.cmake [hunk 1]
```diff
@@ -225,6 +225,7 @@ else()
     "SHELL:-s EXPORT_ALL=0"
     "SHELL:-s VERBOSE=0"
     "SHELL:-s FILESYSTEM=0"
+    "SHELL:-s INCOMING_MODULE_JS_API=[preRun,locateFile,arguments,onExit,wasmMemory,buffer,instantiateWasm,mainScriptUrlOrBlob]"
```
Comment (fs-eire) [inline cmake/onnxruntime_webassembly.cmake:228]: do we need to use the same list for ort-web to onnxruntime_test_all ?

Comment (fs-eire) [reply cmake/onnxruntime_webassembly.cmake:228]: "mainScriptUrlOrBlob", for example, is not used by onnxruntime_test_all but it is required by ort-wasm. So perhaps need to figure out the full list of INCOMING_MODULE_JS_API per target.

Comment (guschmue) [reply cmake/onnxruntime_webassembly.cmake:228]: I can trim things down once ci is happy.

**Review by mszhanyi — APPROVED** [review]

**Review by snnn — COMMENTED** [review]
Please also update https://github.com/microsoft/onnxruntime/blob/main/cgmanifests/generated/cgmanifest.json with this new commit id.

**Pull request closed without merging**